test/integration/profile_test.rb

```diff
@@ -133,6 +133,19 @@ def sign_out
     assert page.has_link?("@nick1", href: "https://twitter.com/nick1")
   end
 
+  test "adding X(formerly Twitter) username without filling in your password" do
```

Comment:
Note to self: I need to write the test in the system folder https://github.com/rubygems/rubygems.org/blob/dd89efd65599dc798ae50367d3c52034646a1563/test/system/multifactor_auths_test.rb#L147